src/mango/src/mango_cursor_view.erl

```diff
@@ -35,6 +35,19 @@
 
 -define(HEARTBEAT_INTERVAL_IN_USEC, 4000000).
 
+% viewcbargs wraps up the arguments that view_cb uses into a single
+% entry in the mrargs.extra list. We use a Map to allow us to later
+% add fields without having old messages causing errors/crashes.
+viewcbargs_new(Selector, Fields) ->
+    #{
+        selector => Selector,
+        fields => Fields
+    }.
+viewcbargs_get(selector, Args) when is_map(Args) ->
```

Comment:
Another alternative we typically use is `viewcbargs_get(selector, #{} = Args)` but either way if fine.

Reply:
I'm going to leave this one as is for now. Let me know if you'd rather I change it.